docroot/sites/default/settings.php

```diff
@@ -17,7 +17,8 @@
 $trusted_hosts = getenv('TRUSTED_HOSTS', true);
 
 $settings['trusted_host_patterns'] = [
-  $trusted_hosts
+  $trusted_hosts,
+  '^localhost$', // For debugging purposes, see https://dsdmoj.atlassian.net/wiki/spaces/HUB/pages/3790667855/Drupal+and+JSON+API#Viewing-JSON%3AAPI-on-Cloud-platform-(debugging)
```

Comment:
Assuming this is okay from a security standpoint.  As the only way to use localhost would be to port forward in.

Reply:
I think so..